tests/Platforms/AbstractPlatformTestCase.php

```diff
@@ -239,35 +239,20 @@ public function testGeneratesTableAlterationSql(): void
         $table = new Table('mytable');
         $table->addColumn('id', 'integer', ['autoincrement' => true]);
         $table->addColumn('foo', 'integer');
-        $table->addColumn('bar', 'string', ['length' => 32]);
-        $table->addColumn('bloo', 'boolean');
+        $table->addColumn('bloo', 'integer', ['notnull' => false]);
         $table->setPrimaryKey(['id']);
 
-        $tableDiff                        = new TableDiff('mytable');
-        $tableDiff->fromTable             = $table;
-        $tableDiff->newName               = 'userlist';
-        $tableDiff->addedColumns['quota'] = new Column('quota', Type::getType('integer'), ['notnull' => false]);
-        $tableDiff->removedColumns['foo'] = new Column('foo', Type::getType('integer'));
-        $tableDiff->changedColumns['bar'] = new ColumnDiff(
-            new Column(
-                'baz',
```

Comment:
This part of the test is invalid.

First of all, the column in `$changedColumns` cannot have its name changed. The old and the new column should have the same name to be added to `$changedColumns`: 
https://github.com/doctrine/dbal/blob/58d4b3c78988dec39294a4508cd3272265e065db/src/Schema/Comparator.php#L205-L225

Second, the test expects to a rename column SQL to be generated for MySQL: https://github.com/doctrine/dbal/blob/08630c6188d70b0f7a98b9c4a079f5f7a76b6cba/tests/Platforms/AbstractMySQLPlatformTestCase.php#L64

But not on Oracle: https://github.com/doctrine/dbal/blob/4bb2c0b2bf735639b1e32796e506f8be9fcac7f7/tests/Platforms/OraclePlatformTest.php#L53